torax/_src/geometry/geometry_loader.py

```diff
@@ -25,6 +25,8 @@
 import torax
 
 
+from torax.torax_imastools.util import load_IMAS_data
```

Comment:
import the module here rather than the `load_IMAS_data` object https://google.github.io/styleguide/pyguide.html#22-imports